src/main/flight/pid.c

```diff
@@ -436,28 +437,71 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float calcHorizonLevelStrength(void)
 // The impact is possibly slightly slower performance on F7/H7 but they have more than enough
 // processing power that it should be a non-issue.
 STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_t *pidProfile, const rollAndPitchTrims_t *angleTrim,
-                                                        float currentPidSetpoint, float horizonLevelStrength)
+                                                        float rawSetpoint, float horizonLevelStrength, bool newRcFrame)
 {
     const float levelAngleLimit = pidProfile->levelAngleLimit;
     // calculate error angle and limit the angle to the max inclination
     // rcDeflection is in range [-1.0, 1.0]
-    float angle = levelAngleLimit * getLevelModeRcDeflection(axis);
+    float angleTarget = levelAngleLimit * getLevelModeRcDeflection(axis);
 #ifdef USE_GPS_RESCUE
-    angle += gpsRescueAngle[axis] / 100; // ANGLE IS IN CENTIDEGREES
+    angleTarget += gpsRescueAngle[axis] / 100; // ANGLE IS IN CENTIDEGREES
+#endif
+    const float currentAngle = (attitude.raw[axis] - angleTrim->raw[axis]) / 10.0f;
+    const float angleError = angleTarget - currentAngle;
+    const float angleErrorCorrection = angleError * pidRuntime.levelGain;
+    //angle loop feedforward
+    float angleFeedforward = 0;
+#ifdef USE_FEEDFORWARD
+    if (newRcFrame){
+        const float feedforwardJitterFactor = pidGetFeedforwardJitterFactor();
```

Comment:
since `pidGetFeedforwardJitterFactor()` just returns `pidRuntime.feedforwardJitterFactor`, why not just use `pidRuntime.feedforwardJitterFactor` directly?

Reply:
A get was needed from within feedforward.c, because it does not have direct access to any of the pidRuntime variables.  But in pid.c, all the pidRuntime variables are available, directly, as in the lines below and elsewhere.

Reply:
makes sense. I've made the change.